mne/io/kit/kit.py

```diff
@@ -720,14 +720,17 @@ def get_kit_info(rawfile, allow_unknown_format, standardize_names=None,
                     hsp.append(rr)
 
             # nasion, lpa, rpa, HPI in native space
-            elp = [dig.pop(key) for key in (
-                'fidnz', 'fidt9', 'fidt10',
-                'hpi_1', 'hpi_2', 'hpi_3', 'hpi_4')]
+            elp = []
+            for key in (
+                    'fidnz', 'fidt9', 'fidt10',
+                    'hpi_1', 'hpi_2', 'hpi_3', 'hpi_4'):
+                if key in dig:
+                    elp.append(dig.pop(key))
             if 'hpi_5' in dig and dig['hpi_5'].any():
                 elp.append(dig.pop('hpi_5'))
             elp = np.array(elp)
             hsp = np.array(hsp, float).reshape(-1, 3)
-            assert elp.shape in ((7, 3), (8, 3))
+            assert elp.shape in ((6, 3), (7, 3), (8, 3))
```

Comment:
Now that it's possible for a user (as opposed to only a dev making code chainges) to run into a situation where this could be violated, this should become a proper check and raise, e.g.:
```suggestion
            if elp.shape not in ((6, 3), (7, 3), (8, 3)):
                raise RuntimeError(f
                    f'Fewer than 3 HPI coils found, got {len(elp) - 3}')
```

Reply:
Implemented the suggested change.